wire-ios/Wire-iOS/Sources/Helpers/syncengine/ZMConversation+Calling.swift

```diff
@@ -66,9 +66,6 @@ extension ZMConversation {
             if granted {
                 let joined = self.voiceChannel?.join(video: video, userSession: userSession) ?? false
 
-                if joined {
-                    Analytics.shared.tagMediaActionCompleted(video ? .videoCall : .audioCall, inConversation: self)
-                }
```

Comment:
If we no longer need `joined` then we no longer need to store it.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/daaae6bd1616d91be26d560bc121ef41579572cd